src/lib/prepare-template.js

```diff
@@ -0,0 +1,16 @@
+import path from 'path';
+import fs from 'fs-extra';
+
+export async function prepareTemplate(basename, appName) {
```

Comment:
@izqui As you can see, I didn't go with the destruction syntax here any more. The main reason really is that, we keep composing the tasks within the handler where we have full control over what parameters are being passed to library functions.

Happy to go with `options` object with destruction syntax again, if you have strong feelings about that.

Reply:
👍 I think explicitly passing the parameters each function needs is better than the options object.

Reply:
Wonderful!